test/integration/testdata/pilot.yaml.tmpl

```diff
@@ -1,14 +1,53 @@
 # pilot service for discovery
+
+{{if .DlvPort}}
```

Comment:
I will write a separate doc in a separate PR. It is helpful for remote debugging of an existing pilot container. Briefly, it allows to open an extra port to which a remote debugger can attach. It also mounts a volume containing dlv.